docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -319,17 +377,74 @@ variables; worker nodes **must** specify the `CUBESTORE_WORKER_PORT` and
 `CUBESTORE_META_ADDR` environment variables. More information about these
 variables can be found [in the Environment Variables reference][ref-config-env].
 
+A sample Docker Compose stack setting this up might look like:
+
+```yaml
+version: '2.2'
+services:
+  cubestore_router:
+    restart: always
+    image: cubejs/cubestore:latest
+    environment:
+      - CUBESTORE_LOG_LEVEL=trace
+      - CUBESTORE_SERVER_NAME=cubestore_router:9999
+      - CUBESTORE_META_PORT=9999
+      - CUBESTORE_WORKERS=cubestore_worker_1:9001,cubestore_worker_2:9001
+    expose:
+      - 9999 # This exposes the Metastore endpoint
+      - 3306 # This exposes the MySQL endpoint
+      - 3030 # This exposes the HTTP endpoint for CubeJS
+  cubestore_worker_1:
+    restart: always
+    image: cubejs/cubestore:latest
+    environment:
+      - CUBESTORE_SERVER_NAME=cubestore_worker_1:9001
+      - CUBESTORE_WORKER_PORT=9001
+      - CUBESTORE_META_ADDR=cubestore_router:9999
+    depends_on:
+      - cubestore_router
+    expose:
+      - 9001
+  cubestore_worker_2:
+    restart: always
+    image: cubejs/cubestore:latest
+    environment:
+      - CUBESTORE_SERVER_NAME=cubestore_worker_2:9001
+      - CUBESTORE_WORKER_PORT=9002
+      - CUBESTORE_META_ADDR=cubestore_router:9999
+    depends_on:
+      - cubestore_router
+    expose:
+      - 9001
+
+  cube:
+    image: cubejs/cube:latest
+    ports:
+      - 4000:4000
+      - 9999
+    environment:
+      - CUBEJS_CUBESTORE_HOST=cubestore_router
+      - CUBEJS_CUBESTORE_PORT=9999
+    depends_on:
+      - cubestore_router
+    volumes:
+      - .:/cube/conf
+```
+
 ### Storage
 
 <!-- prettier-ignore-start -->
 [[warning | ]]
 | Cube Store can only use one type of remote storage at runtime.
 <!-- prettier-ignore-end -->
 
+The storage layer in Cube Store is built from the ground-up to take full
```

Comment:
@hassankhan I'm not sure it is technically correct. I'd probably rephrase it and share that Cube Store stores data parquet and it preferably to use storage like S3.